### ISSUE TITLE
ui: add 'last cleared' timestamp to statement list on table detail page

### ISSUE THREAD

**Issue by Annebirzin** [cockroachdb/cockroach#45479]
- [ ] add 'last cleared' timestamp to statement list on table detail page
![Screen Shot 2020-02-26 at 5 44 38 PM](https://user-images.githubusercontent.com/51920721/75395109-acd72480-58bf-11ea-8777-5707af43b098.png)

**Comment by dhartunian** [comment]
@Annebirzin can we set this ticket aside along with #45480?

**Comment by Annebirzin** [comment]
@dhartunian yep, both of these apply to statements tab on table detail page which was removed
